### ISSUE TITLE
continue game killed my attempt at playing the beta

### ISSUE THREAD

**Issue by ilanpo** [nicupavel/openpanzer#234]
so when i tried to play the british campaign i played through some missions then quit the game, (i do not remember if i manual saved or not), when i relaunched the game and pressed the continue game button on the main menu the year glitched out into a state of NaN or None depending where it was displayed. and it semingly unlocked all units from every single year for me to buy,
this ruined my play-through (off-course) since when i wanted to buy units since i have to now remember what units are available. nicu plz fix 
(P.S its probably not campaign specific as it already fucked up my german world campaign too exactly same details as with the brit's)
-tldr: continue game lets you time travel

**Comment by nicupavel** [comment]
Would be helpful if we'd know what platform (android/web) you are playing on.
